osm2gtfs/tests/tests_accra.py

```diff
@@ -158,9 +158,14 @@ def test_gtfs_from_cache(self):
                          "Wrong routes count in the generated GTFS")
 
 
-def suite():
-    tests = ['test_refresh_routes_cache', 'test_refresh_stops_cache', 'test_gtfs_from_cache']
-    return unittest.TestSuite(map(TestAccra, tests))
+def load_tests(loader, tests, pattern):
+    # pylint require to use the variables
+    loader = loader
+    tests = tests
+    pattern = pattern
+    test_cases = ['test_refresh_routes_cache', 'test_refresh_stops_cache', 'test_gtfs_from_cache']
```

Comment:
To me this line seems to be very close to the individual needs of Accra, and I guess other networks and other tests regarding some core components might need to have another logic for the test cases. I would like to see them living in the individual test suite.

I'm not a unit test expert in Python! And maybe I'm on a wrong track, question: wouldn't it better/easier to have one "defined and initial" test case for all kind of types of tests (we could call it `test_summary` or anything else), basically a wrapper around all the test cases this is implementing. This will allow other test suits, that are very different (e.g. testing core components will be different than testing a creator set), to implement their test cases, also in a certain order, etc...

Reply:
Sorry, this is already in `tests_accra.py`. So never mind this comment.